src/collections/news/2020-05-28-layer5-and-hasicorp-launch-service-mesh-partnership/index.mdx

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "Layer5 and HashiCorp Launch Service Mesh Partnership"
+date:   2020-05-28 00:00:00 +08000
+author: Lee Calcote
+type: news 
+permalink: /company/news/layer5-hashicorp-launch-service-mesh-partnership
+image: ./2020-05-28-layer5-hashicorp-launch-service-mesh-partnership/layer5-hashicorp.png
+---
+
+import { NewsWrapper } from "../News.style.js";
+
+<NewsWrapper>
+
+_Announced on May 28th, 2020 at DockerCon Live 2020._
+
+## Meshery Announces Native Support for Consul Service Mesh
+
+Today, we are pleased to announce the technology partnership of Layer5 and HashiCorp. Layer5’s [Meshery](https://layer5.io/meshery), the service mesh management plane, and HashiCorp’s [Consul](https://consul.io) integrate to provide advanced, cloud native infrastructure solutions for containerized and non-containerized workloads - strengths of Consul. The marriage of these technologies forms a layering of network planes: data, control, and management.
+
+<a href="./2020-05-28-layer5-hashicorp-launch-service-mesh-partnership/Layer5-HashiCorp-Service-Mesh-Partnership.png" alt="Layer5 HashiCorp Service Mesh Partnership"><img src="./2020-05-28-layer5-hashicorp-launch-service-mesh-partnership/Layer5-HashiCorp-Service-Mesh-Partnership.png" alt="Layer5 HashiCorp Service Mesh Partnership" alt="Layer5 HashiCorp Service Mesh Partnership" width="100%" /></a>
```

Comment:
the `a` tag should direct the user to the image that lives in the same folder of this `index.mdx`
remove the `alt` It is not needed for the `a` tag
for the image `import` the images as `Image1` or something like that and use it in the `src` of the `img` tag.
also there is no need of two `alt` in `img` tag